file 5.40 now also print the amount of pages in a PDF while the tests have not caught up yet

diff --git a/test/test.py b/test/test.py
index 949c77e..983837a 100755
--- a/test/test.py
+++ b/test/test.py
@@ -107,7 +107,7 @@ class MagicTest(unittest.TestCase):
         try:
             self.assert_values(m, {
                 'magic._pyc_': 'python 2.4 byte-compiled',
-                'test.pdf': 'PDF document, version 1.2',
+                'test.pdf': 'PDF document, version 1.2, 2 pages',
                 'test.gz':
                     ('gzip compressed data, was "test", from Unix, last '
                      'modified: Sun Jun 29 01:32:52 2008',
